test/e2e/azure_test.go

```diff
@@ -68,53 +68,69 @@ var _ = Describe("Workload cluster creation", func() {
+						Namespace:             namespace,
+						ClusterName:           clusterName,
+						SkipCleanup:           skipCleanup,
+					}
+				})
+			})
+
 		})
 	})
 
-	Context("Creating highly available control-plane cluster", func() {
-		It("With 3 control-plane nodes and 2 worker nodes", func() {
-			cluster, _, _ = clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
-				ClusterProxy: bootstrapClusterProxy,
-				ConfigCluster: clusterctl.ConfigClusterInput{
-					LogFolder:                filepath.Join(artifactFolder, "clusters", bootstrapClusterProxy.GetName()),
-					ClusterctlConfigPath:     clusterctlConfigPath,
-					KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
-					InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
-					Flavor:                   clusterctl.DefaultFlavor,
-					Namespace:                namespace.Name,
-					ClusterName:              clusterName,
-					KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
-					ControlPlaneMachineCount: pointer.Int64Ptr(3),
-					WorkerMachineCount:       pointer.Int64Ptr(2),
-				},
-				CNIManifestPath:              e2eConfig.GetVariable(CNIPath),
-				WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
-				WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
-				WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+	Describe("Creating highly available control-plane cluster", func() {
+		Context("With 3 control-plane nodes and 2 worker nodes", func() {
+			It("deploys the cluster", func() {
+				cluster, _, _ = clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+					ClusterProxy: bootstrapClusterProxy,
+					ConfigCluster: clusterctl.ConfigClusterInput{
+						LogFolder:                filepath.Join(artifactFolder, "clusters", bootstrapClusterProxy.GetName()),
+						ClusterctlConfigPath:     clusterctlConfigPath,
+						KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+						InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+						Flavor:                   clusterctl.DefaultFlavor,
+						Namespace:                namespace.Name,
+						ClusterName:              clusterName,
+						KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
+						ControlPlaneMachineCount: pointer.Int64Ptr(3),
+						WorkerMachineCount:       pointer.Int64Ptr(2),
+					},
+					CNIManifestPath:              e2eConfig.GetVariable(CNIPath),
+					WaitForClusterIntervals:      e2eConfig.GetIntervals(specName, "wait-cluster"),
+					WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
+					WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
+				})
 			})
 
-			Context("Creating a accessible load balancer", func() {
+			It("Creating a accessible load balancer", func() {
```

Comment:
```suggestion
			It("Creates an accessible load balancer", func() {
```

is this more readable / grammatically correct?

Reply:
good catch.  now that it is closer to a common ginko structure the `It` clauses the tests will read better with updates like this. Thanks!